LICENSE

```diff
@@ -1,15 +1,22 @@
-Fluent UI React
-
-Copyright (c) Microsoft Corporation
-
-All rights reserved.
-
 MIT License
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31560&commitId=f4afcbfa32500e87b96cc65bb3cad88b0d5a8b07&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>